cli/src/bootstrap_organization.rs

```diff
@@ -11,9 +11,6 @@ use libparsec::{
 
 #[derive(Args)]
 pub struct BootstrapOrganization {
-    /// Parsec config directory
-    #[arg(short, long)]
-    config_dir: Option<PathBuf>,
```

Comment:
We don't need the `config-dir` here because the CLI only interact with the server ?

Reply:
Nope, this is because I didn't handle it for the moment